packages/node/src/plugins/NodeFederationPlugin.ts

```diff
@@ -141,9 +142,12 @@ class NodeFederationPlugin {
 
     const pluginOptions = {
       ...this._options,
-      remotes: parseRemotes(
-        this._options.remotes || {}
-      ) as ModuleFederationPluginOptions['remotes'],
+      // @ts-ignore
+      remotes: !this.experiments.disableRemoteSideloader
+        ? (parseRemotes(
```

Comment:
Parse remote causes many limits and everything is promise and manually allowing for many fs implementation in one promise template is cumbersome. All management and capability for externals is restored to webpack now.